configs/dsdl/Readme.md

```diff
@@ -0,0 +1,87 @@
+# DSDL Detection Dataset
+
+## 1. What is DSDL ?
+
+Data is the cornerstone of artificial intelligence. The efficiency of data acquisition, exchange, and application directly impacts the advances in technologies and applications. Over the long history of AI, a vast quantity of data sets have been developed and distributed. However, these datasets are defined in very different forms, which incurs significant overhead when it comes to exchange, integration, and utilization -- it is often the case that one needs to develop a new customized tool or script in order to incorporate a new dataset into a workflow.
+
+To overcome such difficulties, we develop **Data Set Description Language (DSDL)**. More details please visit our [official documents](https://opendatalab.github.io/dsdl-docs/getting_started/overview/).
+
+## 2. How to train a dsdl dataset with mmdetection.
+
+- install dsdl:
+
+  install by pip:
+
+  ```
+  pip install dsdl
+  ```
+
+  install by source code:
+
+  ```
+  git clone https://github.com/opendatalab/dsdl-sdk.git -b dev-alg
+  cd dsdl-sdk
+  python setup.py install
+  ```
+
+- install mmdet and pytorch:
+  please refer this [installation documents](https://mmdetection.readthedocs.io/zh_CN/3.x/get_started.html).
```

Comment:
This is an English doc, please link to the English version of installation guide.